### ISSUE TITLE
Fetch details of user from discourse for participant page.

### ISSUE THREAD

**Issue by nemaniarjun** [AICrowd/AIcrowd#832]

**Comment by nemaniarjun** [comment]
This was made to add the posts made by a user on discourse, to the profile on AIcrowd.
@spMohanty do we need this?

Ref: https://aicrowd-design.netlify.com/template-user-posts

**Comment by spMohanty** [comment]
Yes, we still do need this. Lets see if theres an easy way to do this using the Discourse  APIs.

**Comment by nemaniarjun** [comment]
Cant find any way to do this on SO and the docs.

**Comment by spMohanty** [comment]
@nemaniarjun : Maybe https://github.com/discourse/discourse_api ?

**Comment by nemaniarjun** [comment]
# Get Comments
https://discourse.aicrowd.com/user_actions.json?username=arjun_nemani&filter=5
# Get Posts and Comments
https://discourse.aicrowd.com/user_actions.json?username=arjun_nemani&filter=4,5

**Comment by nemaniarjun** [comment]
Do we show only the posts? or Both? @jsnrynlds Can you check the design as per the jsons above?

**Comment by jsnrynlds** [comment]
Good question @nemaniarjun. I think we can show only the comments/posts but with the topic title above.

Will this be styled as per [template designs](https://aicrowd-design.netlify.com/template-user-posts) or using Discourse?

**Comment by nemaniarjun** [comment]
We would need to come up with the design.
